src/components/ha-alert.ts

```diff
@@ -60,6 +60,7 @@ class HaAlert extends LitElement {
             <slot></slot>
           </div>
           <div class="action">
+            <slot name="action"></slot>
```

Comment:
This is so much better and should really just replace all uses of `actionText` in the future

Reply:
I'll do a cleanup after this PR is merged, it's currently only used in the supervisor and the gallery

Reply:
<https://github.com/home-assistant/frontend/pull/10646>